content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,259 @@
+- :guilabel:`Department`: Select the employee's department from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected department's details. Modifications to the
+  :guilabel:`Department Name`, :guilabel:`Parent Department`, :guilabel:`Manager`, and
+  :guilabel:`Company` can be made here. Click :guilabel:`Save` after making edits.
+- :guilabel:`Manager`: Select the employee's :guilabel:`Manager` from the drop-down menu. The
+  :guilabel:`External Link` button opens the selected manager's :guilabel:`Employee Form`.
+  Modifications to the form can be made here. Click :guilabel:`Save` after any edits are made.
+- :guilabel:`Coach`: Select the employee's :guilabel:`Coach` from the drop-down menu. The external
+  link button opens the selected coach's :guilabel:`Employee Form`. Modifications to the form can be
+  made here. Click :guilabel:`Save` after any edits are made.
+
+ .. note::
+    After a :guilabel:`Manager` is selected, if the :guilabel:`Coach` field is blank, the selected
+    manager automatically populates the :guilabel:`Coach` field.
+
+Additional information tabs
+===========================
+
+Resume tab
+----------
+
+Resume
+~~~~~~
+
+Next, the employee's work history is entered in the :guilabel:`Resume` tab. Each previous experience
+must be entered individually. Click :guilabel:`Create a New Entry`, and the :guilabel:`Create Resume
+lines` form appears. Enter the following information, then click the :guilabel:`Save & Close` button
+if there is only one entry to add, or click the :guilabel:`Save & New` button to save the current
+entry and create another line.
+
+.. image:: new_employee/resume-lines.png
+   :align: center
+   :alt: Add information for the previous work experience in this form.
+
+- :guilabel:`Name`: Type in the :guilabel:`Name` of the previous work experience.
+- :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,
+  :guilabel:`Education`, :guilabel:`Internal Certification`, :guilabel:`Internal Training`, or type
+  in a new entry.
+- :guilabel:`Display Type`: Select either :guilabel:`Classic`, :guilabel:`Certification`, or
+  :guilabel:`Course` from the drop-down menu.
+- :guilabel:`Date Start` and :guilabel:`Date End`: Enter the start and end dates for the work
+  experience. To select a date, use the left and right arrows to scroll to the desired month, then
+  click on the day to select it.
+- :guilabel:`Description`: Enter any relevant details in the description field.
+
+Skills
+~~~~~~
+
+An employee's skills can be entered in the :guilabel:`Resume` tab in the same manner a resume line
+is created. Click the :guilabel:`Create a New Entry` button under *Skills* and a :guilabel:`Create
+Skills` form appears. Fill in the information, then click the :guilabel:`Save & Close` button if
+there is only one entry to add, or click the :guilabel:`Save & New` button to save the current entry
+and immediately create a new entry.
```

Comment:
```suggestion
there is only one entry to add, or click the :guilabel:`Save & New` button to save the current
entry and immediately create a new entry.
```
100th character line break

Reply:
I double checked this with Sam and this looks correct- it hits right at 100 character mark.